avocado/utils/kernel.py

```diff
@@ -72,7 +72,8 @@ def download(self):
         full_url = self.URL + self.SOURCE.format(version=self.version)
         self.asset_path = asset.Asset(full_url, asset_hash=None,
                                       algorithm=None, locations=None,
-                                      cache_dirs=self.data_dirs).fetch()
+                                      cache_dirs=[self.work_dir],
+                                      expire=None).fetch()
```

Comment:
Hello @clebergnu, can I ask about the motvation? What was broken here? In my opinion it was actually better before, because the `kernel.py` accepts `data_dirs` argument, or it uses `[self.work_dir]` instead. So I think the previous version was actually correct one.